specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```diff
@@ -1085,6 +1112,14 @@
           },
           {
             "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "unassignJobRequest",
+            "description": "Request body for unassign route",
+            "schema": {
+              "$ref": "#/definitions/UnassignJobRequest"
+            }
```

Comment:
Is the request body truly optional? What is the behavior if it is not provided?

Reply:
For this particular route of unassign job, there is a single optional property that `UnassignJobRequest` wraps: 'bool? SuspendMatching'. If an empty body is provided, then internally, the optional value default to false (which is the default behaviour of JobRouter in the first place).

Similar behavior exists in other routes which accepts optional body, for e.g., /routing/jobs/{id}:cancel